python/tank/bootstrap/manager.py

```diff
@@ -752,11 +836,12 @@ def _cache_apps(self, tk, config_engine_name, config_entity, progress_callback,
             progress_value = 0.4 + idx * (0.3 / len(descriptors))
 
             if not descriptor.exists_local():
-                message = "Downloading %s (%s of %s)..." % (descriptor, idx+1, len(descriptors))
+                message = "Downloading %s (%s of %s)..." % (descriptor, idx + 1, len(descriptors))
                 self._report_progress(progress_callback, progress_value, message)
                 descriptor.download_local()
             else:
-                message = "%s already installed locally (%s of %s)." % (descriptor, idx+1, len(descriptors))
+                message = "%s already installed locally (%s of %s)." % (descriptor, idx + 1, len(descriptors))
```

Comment:
Let's address this in the progress reporting ticket for the desktop, shall we? We'll be in a better position there to decide how we want to report everything.